packages/itwinui-css/src/table/sizes.scss

```diff
@@ -17,8 +21,22 @@
     .iui-table-header .iui-table-cell {
       padding-block: calc(var(--iui-size-s) * 0.25);
     }
+
+    .iui-table-cell {
+      padding-left: calc(var(--iui-size-2xs));
```

Comment:
I think we'll want the spacing to the left & right of columns to be the same.  On the far right side of the table, if a scrollbar is present, [the scrollbar is `var(--iui-size-xs)` wide](https://github.com/iTwin/iTwinUI/blob/f8a0c8aee62cc848094d3efa4302415d159a4756/packages/itwinui-css/src/utils/mixins.scss#L41).  I'm unsure if we'll want the left/right padding to be any less than that.